spec.html

```diff
@@ -11516,7 +11523,7 @@ <h1>Realms</h1>
             [[LoadedModules]]
           </td>
           <td>
-            a List of Records with fields [[Specifier]] (a String) and [[Module]] (a Module Record)
+            a List of Records with fields [[Specifier]] (a String), [[Attributes]] (a List of ImportAttribute Records) and [[Module]] (a Module Record)
```

Comment:
```suggestion
            a List of Records with fields [[Specifier]] (a String), [[Attributes]] (a List of ImportAttribute Records), and [[Module]] (a Module Record)
```
Oxford comma.

However, this Record schema shows up 3 times, so I think it'd be preferable to 'declare' it (give it a name and list its fields in a table).